packages/palette/scripts/new_icons_script.rb

```diff
@@ -44,22 +47,22 @@ def fix_attrs(node)
   node.children.each { |child| fix_attrs(child) }
 end
 
-def format(slug, title)
+def format(slug)
   input_path = "new_icons/output/min/#{slug}.min.svg"
   xml = File.read(input_path)
   doc = Nokogiri::XML(xml)
   svg = doc.at_css('svg')
-  title = "<title>#{title}</title>"
+  title = "<title>{title}</title>"
```

Comment:
Assuming that `title` was never used as an argument and this is just for interpolation later?

Reply:
Title was previously used as an argument, but we no longer want to interpolate at the script level. `<title>{title}</title>` is the actual JSX we want to produce. The title value comes from the prop `title` and is given a default value of whatever the provided title is.